src/sb_utils.R

```diff
@@ -1,4 +1,14 @@
-sb_replace_files <- function(sb_id, ..., file_hash){
+
+#' @param use_task_table logical specifying whether to call `do_item_replace_tasks`
+#' which will create a task_table of the files to push to ScienceBase. This prevents them
+#' all from failing if one fails. Defaults to `TRUE`.
+#' 
+sb_replace_files <- function(sb_id, ..., file_hash, use_task_table = TRUE){
+  
+  files <- c(...)
+  
+  # Throw error if there are no files given to push
+  stopifnot(length(files) > 0 | !missing(file_hash))
   
   if (!sbtools::is_logged_in()){
```

Comment:
I think so be safe, you probably need to put this ahead of each task call to upload the tiles. Long-running uploads may timeout the session, and then all of the following tasks will fail.